src/module/item/sheet/base.ts

```diff
@@ -379,6 +383,16 @@ export class ItemSheetPF2e<TItem extends ItemPF2e> extends ItemSheet<TItem> {
                 this.item.update({ [`system.variants.-=${index}`]: null });
             });
         }
+
+        for (const section of htmlQueryAll(html, "[data-action=edit-section]")) {
+            section.addEventListener("click", (event) => {
+                if (!(event.target instanceof HTMLElement)) return;
```

Comment:
Is this not the same element as `section`?

Reply:
It's _supposed_ to be the little 🗒️ icon in the header. I might be able to attach the listener to the section instead if that's better.

Reply:
Moved the listener to the container instead.